cxxheaderparser/types.py

```diff
@@ -298,6 +301,23 @@ def format_decl(self, name: str):
         return f"{c}{v}{self.typename.format()} {name}"
 
 
+@dataclass
+class Operator:
```

Comment:
I think if this isn't exposed to users, it probably doesn't belong in `types.py`

Reply:
ok,
I had thought about it before, but since this module contains a lot of data classes, I decided to put it there.
I removed it from `types.py` and moved to `parser.py`, because only there it is used (I personally prefer to keep many small files, but I see that convention in this project is to keep small number of modules).